Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java

```diff
@@ -116,9 +116,9 @@ static boolean isStartMutedNotConfigured(@NonNull final Map<String, String> extr
 
     public static class Builder {
         private static final String EXTRA_START_MUTED_KEY = "startMuted";
-        private static final String EXTRA_SOUND_ENABLED_KEY = VungleInterstitial.SOUND_ENABLED_KEY;
-        private static final String EXTRA_ORDINAL_VIEW_COUNT_KEY = VungleInterstitial.ORDINAL_VIEW_COUNT_KEY;
-        private static final String EXTRA_ORIENTATION_KEY = VungleInterstitial.AD_ORIENTATION_KEY;
+        private static final String EXTRA_SOUND_ENABLED_KEY = "vungleSoundEnabled";
```

Comment:
We also need to remove this line and related code parts `withSoundEnabled()`, thanks.